src/files/selected-actions/SelectedActions.stories.js

```diff
@@ -15,12 +18,8 @@ storiesOf('Files', module)
         share={action('Share Files')}
         download={action('Download Files')}
         rename={action('Rename Files')}
-        inspect={action('Inspect Files')}
-      />
-    </div>
-  ))
-  .add('Actions for Multiple Files', () => (
-    <div className='ma2'>
+        inspect={action('Inspect Files')} />
+
       <SelectedActions
```

Comment:
Why merge these into 1 story? Seems more natural to have a story per state, or 1 story with knobs on so the user can try out different states https://github.com/storybooks/storybook/tree/master/addons/knobs